modules/network-observability-netobserv-cli-install.adoc

```diff
@@ -18,24 +18,23 @@ You can optionally use Krew to install the `netobserv` CLI plugin. For more info
 
 .Procedure
 
-. Download the link:https://mirror.openshift.com/pub/cgw/netobserv/latest/[`oc netobserv` CLI tar file] that corresponds with your architecture.
```

Comment:
this link is still valid for people to peek at exact naming of these files for different archs, we should still keep it, except now it's no longer a tar file.

Reply:
Indeed. What about:
```
Download the link:https://mirror.openshift.com/pub/cgw/netobserv/latest/[`oc netobserv` file] that corresponds with your architecture.
```